core/src/main/java/org/jdbi/v3/core/statement/StatementExceptions.java

```diff
@@ -136,6 +136,6 @@ public String apply(StatementException exc) {
     }
 
     private static String limit(String s, int len) {
-        return s.substring(0, Math.min(len, s.length()));
+        return s.substring(0, Math.min(len, s.length())) + (s.length() > len ? "[...]" : "");
```

Comment:
Call me a nitpick, but I'm wondering if it might be worth doing this more verbosely, to stay in line with the clarity of code we generally try to achieve by keeping the number of operations per line low?

```
limited = limit(...);
isTruncated = limited.length < str.length;
if (isTruncated) {limit = limit + "[...]"};
```

Reply:
Splitting it into multiple lines I agree with, but adding another method seems overkill and overly verbose, especially considering that both parts contextually are part of the whole idea of 'truncating/limiting'.

Reply:
oh yes, sorry, the `limit` part should be `substring`. No new method of course, just splitting it into lines and a variable/condition :)